accelerator/cuda_accelerator.py

```diff
@@ -167,7 +154,11 @@ def amp(self):
         return None
 
     def is_available(self):
-        return torch.cuda.is_available()
+        try:
+            import torch
+            return torch.cuda.is_available()
+        except ImportError:
+            return True
```

Comment:
Two questions here:
1. Why not `import torch.cuda`?
2. Why return `True` if import fails?